File: Batch_22-26/Task-1/Siddharth_CS/webwindow/app/src/main/java/com/example/webwindow/MainActivity.java
package com.example.webwindow;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.View;
import android.widget.TextView;
import android.widget.Toast;

import com.google.android.material.button.MaterialButton;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        TextView username = (TextView) findViewById(R.id.username);
        TextView password = (TextView) findViewById(R.id.password);

        MaterialButton login = (MaterialButton) findViewById(R.id.login);

        login.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                if(username.getText().toString().equals("admin") && password.getText().toString().equals("admin"))
                    //correct
                    Toast.makeText(MainActivity.this,  "LOGIN SUCCESSFUL", Toast.LENGTH_SHORT ).show();
                    else
                        //incorrect
                    Toast.makeText(MainActivity.this,  "LOGIN FAILED !!!", Toast.LENGTH_SHORT ).show();
            }
        });

    }
}

File: Batch_22-26/Task-2/Parth_36/app/src/main/java/com/example/loginpage/MainActivity.kt
package com.example.loginpage

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.Button
import android.widget.Toast

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        var btn1 :Button = findViewById(R.id.button1)
        val duration = Toast.LENGTH_SHORT
        btn1.setOnClickListener {
            Toast.makeText(this,"Login Success",duration).show()
        }
    }
}